docs/mkdocs/docs/problems.md

```diff
@@ -0,0 +1,198 @@
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/retry/retry_unary.py", line 144, in retry_target
+          result = target()
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/timeout.py", line 120, in func_with_timeout
+          return func(*args, **kwargs)
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/grpc_helpers.py", line 78, in error_remapped_callable
+          raise exceptions.from_grpc_error(exc) from exc
+      google.api_core.exceptions.ResourceExhausted: 429 Resource has been exhausted (e.g. check quota).
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  Script candidate 0: successfully rendered the image
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      [ 2 / 20 ] inspect this; 162s
+        [Show] /:generated-case [162s]
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  Script candidate 0: successfully rendered the image
+      INFO:  So far 2 valid script candidates
+      INFO:  Attempting to select the best script by comparing images
+      INFO:  Selected the best fit image: 2
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      [ 1 / 23 ] inspect this; 168s
+        [Show] /:generated-case [168s]
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  DONE: inspect: this: 168.58s
+      Aborted.
+
+      DEBUG:root:Command error: WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+      E0000 00:00:1735443942.784201    9830 init.cc:229] grpc_wait_for_shutdown_with_timeout() timed out.
+
+      DEBUG:root:Command return code: 1
+```
+
+1. Do not pollute output with traces.
+   - In order to not loose details (to file a ticket) verbose log could be saved to file for convenience.
+
+## Failed to summarize
+
+```
+⬢ [Docker] ❯ pc -q --no-ansi inspect -V /pub/robotics/parts/gobilda:structure/u_channel_2
+Aborted.
+```
+
+```
+⬢ [Docker] ❯ pc --no-ansi inspect -V /pub/robotics/parts/gobilda:structure/u_channel_2
+INFO:partcad:DONE: InitCtx: /tmp: 0.00s
+ERROR:partcad:Failed to associate the model gemini-1.5-pro with the provider
+INFO:partcad:Summary: Failed to summarize
+Failed to summarize
+INFO:partcad:DONE: inspect: this: 3.96s
+Aborted.
+```
+
+1. Get rid of `Aborted` when `-q` provided.
+2. In case of error when `-q` was provided it's really not clear what happened.
+   - Consider writing log files automatically when `-q` provided.
+   - Consider returning a valid JSON with top level details.
```

Comment:
1. Maybe it still okay to be printed but make sure that it's STDERR.
2. `-q` mode is too quiet, it does not feel right.

Reply:
It feels that I do not understand `-q` intentions, and there is `--json` in other commands but anyway it feels like `-q` still should print something like:

```json
{  
  "error": {  
    "code": "PC-AI-001",  
    "message": "Failed to summarize: AI model association failed",  
    "log_file": "~/.partcad/logs/2024-12-29-123456.log"  
  }  
}  
```